packages/ui-tree-browser/src/TreeBrowser/index.tsx

```diff
@@ -172,6 +176,9 @@ class TreeBrowser extends Component<TreeBrowserProps> {
     return this.getExpanded(this.state, this.props)
   }
 
+  get sortOrder() {
+    return this.props.sortOrder
+  }
```

Comment:
I'm not sure a getter is needed here, you can just use `this.props.sortOrder` where you need it. A getter is useful when you want to add some special logic in it.

Reply:
Yes, you're right. I will delete it.